Scripts/rhinoscript/surface.py

```diff
@@ -2025,14 +2025,15 @@ def SurfaceSphere(surface_id):
     return rc
 
 
-def JoinSurfaces(object_ids, delete_input=False):
+def JoinSurfaces(object_ids, delete_input=False, return_all=False):
```

Comment:
Hi @acormier shouldn't the default value be` return_all=True` to keep the current behavior. Or are you happy to introduce a breaking change? This might break currently working scripts in Rh8

Reply:
Hi @goswinr , no `return_all=False` only returns a single id (string) like the original behavior, not introducing any breaking changes.

Reply:
@acormier You are right! Sorry, didn’t realise that rc was a single item in the old version.